apps/platforms/mac/notes/notes.talon

```diff
@@ -0,0 +1,36 @@
+os: mac
+and app: notes
+-
+new note: key(cmd-n)
+duplicate note: key(cmd-d)
+new folder: key(shift-cmd-n)
+toggle folders: key(alt-cmd-s)
+show main: key(cmd-0)
+show as list: key(cmd-1)
```

Comment:
I might suggest matching the existing finder commands for the view options for consistency.

from finder.talon:
```
icon view: key(cmd-1)
column view: key(cmd-3)
list view: key(cmd-2)
gallery view: key(cmd-4)
```

Reply:
Good point, I renamed a couple of commands here https://github.com/knausj85/knausj_talon/pull/649/commits/6c3df7443ae746c6df4aecae116a2a8c484086f1.

I'm not sure that `show main` and `toggle folders` have a direct match in Finder though.